trino-jdbc/src/main/resources/META-INF/services/java.sql.Driver

```diff
@@ -1 +1 @@
-io.prestosql.jdbc.PrestoDriver
+io.trino.jdbc.PrestoDriver
```

Comment:
We could add a class extending `io.trino.jdbc.PrestoDriver` to ease with migration.